sensors/trajectory_calc.py

```diff
@@ -112,6 +113,11 @@ def get_flywheel_speed(self, distance_to_target: float) -> float:
         
         return  min(config.v0_flywheel_minimum + distance_to_target * config.flywheel_distance_scalar, config.v0_flywheel_maximum)
 
+    def get_height_offset(self, distance_to_target: float) -> float:
```

Comment:
What is the thinking behind this??? What is the logic? Why are we making the distance to the target 1.7% of what it was? VERY STRANGE.

Reply:
Okay... I guess I see. Why we think that this is solution is weird to me... It is very ad-hoc and without much empirical reason. But we are going to add 1.7% of the distance to the height.... Weird...